spring-auto-restdocs-example/pom.xml

```diff
@@ -26,7 +26,6 @@
         <start-class>capital.scalable.restdocs.example.Application</start-class>
         <spring-restdocs.version>1.2.1.RELEASE</spring-restdocs.version>
         <spring-auto-restdocs.version>1.0.9-SNAPSHOT</spring-auto-restdocs.version>
-        <spring-restdocs.version>1.2.1.RELEASE</spring-restdocs.version>
```

Comment:
This would be a downgrade. Spring Boot 1.5.6 comes with `1.1.3.RELEASE` as the managed version, see
https://github.com/spring-projects/spring-boot/blob/a5229f24b8239bdeb7ed3edd78a007e37f10b264/spring-boot-dependencies/pom.xml#L167